addons/pvr.vdr.vnsi/src/VNSIData.cpp

```diff
@@ -809,6 +808,164 @@ PVR_ERROR cVNSIData::GetRecordingEdl(const PVR_RECORDING& recinfo, PVR_EDL_ENTRY
   return PVR_ERROR_NO_ERROR;
 }
 
+int cVNSIData::GetDeletedRecordingsCount()
```

Comment:
1) please put the implementation into a separate commit. 
2) you need to bump vnsi protocol when adding new functions. then check if those functions are available on the backend before using those.